pkg/internal/util/env/term.go

```diff
@@ -30,3 +31,21 @@ func IsTerminal(w io.Writer) bool {
 	}
 	return false
 }
+
+// IsSmartTerminal returns true if the writer w is a terminal AND
+// we think that the terminal is smart enough to use VT escape codes etc.
+func IsSmartTerminal(w io.Writer) bool {
+	if !IsTerminal(w) {
+		return false
+	}
+	// explicitly dumb terminals are not smart
+	if os.Getenv("TERM") == "dumb" {
```

Comment:
this is the output with TERM=dumb

```
Creating cluster "kind" ...
 • Ensuring node image (kindest/node:v1.16.2) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.16.2) 🖼
 • Preparing nodes 📦  ...
 ✓ Preparing nodes 📦
 • Creating kubeadm config 📜  ...
 ✓ Creating kubeadm config 📜
 • Starting control-plane 🕹️  ...
```

Reply:
Right. No spinner, no vt codes.
This is already the fallback.